projects/admin/src/app/circulation/patron/card/card.component.ts

```diff
@@ -25,19 +27,19 @@ import { getBootstrapLevel } from '../../../utils/utils';
 })
 export class CardComponent {
 
+  // COMPONENT ATTRIBUTES =====================================================
```

Comment:
For me this is useless.

Reply:
It allows (only for me?) to have a better visualization of the code. Not needed but, for big component, useful